src/components/CardTalent/index.tsx

```diff
@@ -14,13 +33,26 @@ const TalentCarouselCard = styled.div`
   background-color: ${COLORS.background};
 `;
 
-const CardTalent = ({ title, description }: Props) => {
-  return (
-    <TalentCarouselCard>
-      <h1 style={{ color: COLORS.foregroundPrimary }}>{title}</h1>
-      <p>{description}</p>
-    </TalentCarouselCard>
-  );
-};
+const BasicInfoContainer = styled.div`
+  display: flex;
+  gap: 8px;
+  justify-content: center;
+  padding: 8px;
+
+  > *:not(:last-child) {
+    border-right: 1px solid #222;
+    padding-right: 8px;
+  }
+`;
+
+const IconWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  margin-bottom: 8px;
+`;
+
+const DescriptionContainer = styled.div`
+  flex: 1;
+`;
```

Comment:
Doesn't have to be in this PR, but we should probs share these styled components when we introduce Item cards